test/hemera/unsubscribing.spec.js

```diff
@@ -212,6 +212,15 @@ describe('Unsubscribe NATS topic', function() {
         }
       )
 
+      hemera.add(
```

Comment:
What are you trying to test?

Reply:
how `removeAll` function works with wildcard topic

Reply:
...it should remove all patterns event if they is wildcard topic